node-fs-writefile/cp.js

```diff
@@ -0,0 +1,11 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+const copyFrom = process.argv[2];
+const copyTo = process.argv[3];
+
+try {
+  const copiedContents = await readFile(copyFrom, 'utf8');
+  await writeFile(copyTo, copiedContents, 'utf8');
+} catch (error) {
```

Comment:
This works. However, the code you have written works for text files, but not for binary files. Since you provide the 'utf8' encoding when reading the file, you will only be able to successfully read text files. And you also force the file to 'utf8' when you write.

To make this work for binary files (like images), you all have to do is remove the 'utf8' encoding from both calls!